contrib/options/label/Advanced

```diff
@@ -0,0 +1,26 @@
+#424242 Status: Needs feedback ; Feedback is needed
+#fbc02d Status: In progress ; Work is in progress
+#43a047 Status: Completed ; Work is complete
+#00796b Status: Help wanted ; 
+#880e4f Status: Blocked ; 
+#b0bec5 Status: Stale ; 
+
+#d32f2f Kind: Bug ; 
+#607d8b Kind: Question ; 
+#9c27b0 Kind: Security ; 
+#795548 Kind: Testing ; 
+#c62828 Kind: Breaking ; 
+#0288d1 Kind: Feature ; 
+#2e7d32 Kind: Enhancement ; 
+#f4511e Kind: Maintenance ; 
+#00796b Kind: Improvement ; 
```

Comment:
Should we remove impruvement, in fafour of enhancement - or whats the difference between both?

Reply:
Agree two close in meaning

Reply:
I think that enhancements should be considered as an implementation of a new feature.
Improvements improve already existing things.

Reply:
We use `kind/feature` for new features, `kind/enhancement` for existing features being enhanced. 😁

Reply:
For me, enhancements are changes that add extra functionality to an existing feature.
Improvements are changes that do not add functionality. They are only improving things.

For this reason I think that improvements and enhancements are not synonyms.

Reply:
I still think they are too close to be useful imho

Reply:
Ive already seen multiple repos which use both labels simultaneously. Can't we just keep both?

Reply:
Ok. I'm giving up.